demo/crypto-perf/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM centos:7
+
+ENV http_proxy http://proxy.ir.intel.com:911/
+ENV https_proxy http://proxy.ir.intel.com:911/
+ENV no_proxy http://proxy.ir.intel.com:911
```

Comment:
This won't work outside of Intel.
Let the script `demo/build-image.sh` take care of the proxies.